client/flags.go

```diff
@@ -1,14 +1,20 @@
 package client
 
-import "github.com/spf13/cobra"
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/spf13/cobra"
+)
 
 // nolint
 const (
 	// DefaultGasAdjustment is applied to gas estimates to avoid tx
 	// execution failures due to state changes that might
 	// occur between the tx simulation and the actual run.
-	DefaultGasAdjustment = 1.0
-	DefaultGasLimit      = 200000
+	DefaultGasAdjustment  = 1.0
+	DefaultGasLimit       = 200000
+	GasFlagSimulateString = "simulate"
```

Comment:
I think `GasFlagSimulate` should suffice here, no?

Reply:
Done, thanks